requirements.txt

```diff
@@ -11,6 +11,7 @@ projectq
 qiskit
 requests
 rich
+ruff
```

Comment:
Not needed.  Please remove.  There is a difference between runtime dependencies and testing dependencies.